package.json

```diff
@@ -16,14 +16,17 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node pretest.js",
     "test": "mocha --timeout 20000 --require test/init.js",
-    "posttest": "npm run lint"
+    "posttest": "node posttest.js && npm run lint"
   },
   "dependencies": {
     "async": "^1.5.0",
     "debug": "^2.2.0",
+    "lodash": "^4.17.4",
```

Comment:
This appears to only be used in tests. Should it be in `devDependencies` instead?